### ISSUE TITLE
Self-Signed Vault Cert: hashivault_read successful, lookup fails

### ISSUE THREAD

**Issue by atrepca** [TerryHowe/ansible-modules-hashivault#72]
**Versions:**
ansible: 2.5.4
ansible-modules-hashivault: 3.9.5
hvac: 0.6.0

**Environment variables set:**
VAULT_ADDR=https://my.vault.install:8200/
VAULT_CACERT=/Users/atrepca/.certs/vault-ca.pem

This works fine:
```yml
---
- hosts: localhost
  gather_facts: False
  - name: Read Vault secret - foo
    hashivault_read:
      secret: 'secret/foo'
      key: 'bar'
    register: 'bar'
```
This fails:
```yml
---
- hosts: localhost
  gather_facts: False
  vars:
    bar: "{{ lookup('hashivault', 'secret/foo', 'bar') }}"
  tasks:
    - debug:
        msg: "bar={{ bar }}"
```
With a `CERTIFICATE_VERIFY_FAILED` error:
```shell
fatal: [localhost]: FAILED! => {"msg": "An unhandled exception occurred while templating '{{ lookup('hashivault', 'secret/foo', 'bar') }}'. Error was a <class 'ansible.errors.AnsibleError'>, original message: An unhandled exception occurred while running the lookup plugin 'hashivault'. Error was a <class 'ansible.errors.AnsibleError'>, original message: Error reading vault/secret/foo/bar: Exception: HTTPSConnectionPool(host='my.vault.install', port=8200): Max retries exceeded with url: /v1/secret/foo (Caused by SSLError(SSLError(1, u'[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:590)'),))\n"}
```

If I `export VAULT_SKIP_VERIFY=true`, the `lookup` works as expected.

**Comment by atrepca** [comment]
The issue is with the hashivault lookup plugin, `_get_verify` is only reading the `VAULT_CAPATH` environment variable: [ansible/plugins/lookup/hashivault.py#L62](https://github.com/TerryHowe/ansible-modules-hashivault/blob/77aa207583130b7dac0454f6244beb2f0576ce85/ansible/plugins/lookup/hashivault.py#L62). If I set `VAULT_CAPATH` to the same value as `VAULT_CACERT`, the lookup works.

**Comment by TerryHowe** [comment]
This should be resolved in 3.9.6, thanks!
